docs/mint.json

```diff
@@ -79,82 +79,97 @@
   ],
   "navigation": [
     {
-      "group": "Introduction",
+      "group": "Get Started",
       "pages": [
         "introduction/overview",
-        "getting-started/setup"
+        "getting-started/setup",
+        "design/core-design-principles",
+        "design/core-abstractions"
       ]
     },
     {
       "group": "Concepts",
       "pages": [
-        "design/core-abstractions",
-        "design/core-design-principles",
         {
-          "group": "Abstractions",
+          "group": "Backfills",
+          "icon": "database",
+          "pages": [
+            "orchestration/backfills/overview",
+            "orchestration/backfills/guides"
+          ]
+        },
+        {
+          "group": "Blocks",
+          "icon": "block-brick",
           "pages": [
+            "design/blocks",
             {
-              "group": "Blocks",
+              "group": "Types",
               "pages": [
-                "design/blocks",
+                "design/blocks/callbacks",
+                "design/blocks/conditionals",
+                "design/blocks/data-exporter",
+                "design/blocks/data-loader",
+                "design/blocks/extension",
+                "design/blocks/scratchpad",
+                "design/blocks/sensor",
+                "design/blocks/transformer",
                 {
-                  "group": "Types",
+                  "group": "SQL",
                   "pages": [
-                    "design/blocks/callbacks",
-                    "design/blocks/conditionals",
-                    "design/blocks/data-exporter",
-                    "design/blocks/data-loader",
-                    "design/blocks/extension",
-                    "design/blocks/scratchpad",
-                    "design/blocks/sensor",
-                    "design/blocks/transformer",
-                    {
-                      "group": "SQL",
-                      "pages": [
-                        "design/blocks/sql/trino"
-                      ]
-                    }
+                    "design/blocks/sql/trino"
```

Comment:
The trino doc should actually be together with other database integration.
Many of these databases are supported in SQL block.
<img width="278" alt="image" src="https://github.com/mage-ai/mage-ai/assets/80284865/6f756d77-911b-4e85-afbf-82589efc0ed4">